bot/cogs/spam/spam.py

```diff
@@ -0,0 +1,87 @@
+import os
+import pathlib
+import time
+
+import discord
+from discord.ext import commands
+
+from ..variables import *
+
+spam_log_file = str(
+    pathlib.Path(
+        os.path.join(
+            os.path.abspath(os.getcwd()), "bot", "cogs", "spam", "spam_log.txt"
+        )
+    ).resolve()
+)
+
+
+class Spam(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.author.id == self.client.user.id:
+            return
+
+        ctx = await self.client.get_context(message)
+
+        _timestamp = str(int(time.time()))
+        _authorid = str(message.author.id)
+        _channelid = str(message.channel.id)
+
+        _log = _timestamp + "___" + _authorid + "___" + _channelid
+
+        counter_messages = 0
+        counter_channels = 0
+        list_channels = []
+
+        with open(spam_log_file, "r+") as file:
+            file.writelines(f"{str(_log)}\n")
+
+            for line in file.readlines():
+                log_split = line.split("___")
```

Comment:
This file will blow up in length and the bot will be super slow.

Reply:
Again, it's reset every X seconds, so closing this too :)